main.go

```diff
@@ -70,6 +72,7 @@ var (
 	leaderElectionLeaseDuration = flag.Duration("leader-election-lease-duration", 15*time.Second, "Leader election lease duration.")
 	leaderElectionRenewDeadline = flag.Duration("leader-election-renew-deadline", 14*time.Second, "Leader election renew deadline.")
 	leaderElectionRetryPeriod   = flag.Duration("leader-election-retry-period", 4*time.Second, "Leader election retry period.")
+	batchSchedulerName          = flag.String("batchSchedulerName", "", "Use specified scheduler for pods' batch scheduling.")
```

Comment:
Name of the flag should not use camel case. Please change it to `batch-scheduler-name`.